deploy/terraform-aws/node.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash
 
```

Comment:
Use `set -e` here to exit the script when there's an error. This makes debugging the script much easier.